pkg/drivers/generic/metrics.go

```diff
@@ -0,0 +1,46 @@
+package generic
```

Comment:
I'm not sure this file belongs in the generic driver. It doesn't seem to use anything from the generic package, and everything in this file is exported. It should probably be at `pkg/metrics/metrics.go`?

Reply:
It use `Stripped` from the generic package to help print the slow SQL. Or maybe we can move `Stripped` to another general package like `utils`? wdyt

Reply:
Ah right, yeah moving that outside the generic package seems like a good solution to allow reuse without import cycles.